prescient/simulator/master_options.py

```diff
@@ -381,54 +139,13 @@ def _construct_inner_options_parser():
                                         dest='data_directory',
                                         type='string',
                                         default="confcomp_data")
-    guiOverride['--data-directory'] = {}
-    guiOverride['--data-directory']['bpa'] = False
-
-    input_simulation_options.add_option("--plot-individual-generators",
-                                        help="Display stack graph plots showing individual generators, "
-                                             "as opposed to by-type generator behavior. Defaults to False.",
-                                        action="store_true",
-                                        dest="plot_individual_generators",
-                                        default=False)
-    guiOverride["--plot-individual-generators"] = {}
-    guiOverride["--plot-individual-generators"]["--plot-individual-generators"] = False
-
-    input_simulation_options.add_option("--plot-peak-demand",
-                                        help="Generate stack graphs using the specified peak demand, "
-                                             "instead of the thermal fleet capacity. "
-                                             "Defaults to 0.0, indicating disabled.",
-                                        action="store",
-                                        dest="plot_peak_demand",
-                                        type="float",
-                                        default=0.0)
-    guiOverride["--plot-peak-demand"] = {}
-    guiOverride["--plot-peak-demand"]["--plot-peak-demand"] = False
-
-    input_simulation_options.add_option("--disable-plot-legend",
-                                        help="Disable generation of plot legends on individual stack graph plots. "
-                                             "Default to False.",
-                                        action="store_true",
-                                        dest="disable_plot_legend",
-                                        default=False)
-    guiOverride["--disable-plot-legend"] = {}
-    guiOverride["--disable-plot-legend"]["--disable-plot-legend"] = False
-
-    input_simulation_options.add_option('--disable-ruc',
-                                        help='Disables reliability unit commitment during simulation.',
-                                        action='store_true',
-                                        dest='disable_ruc',
-                                        default=False)
-    guiOverride['--disable-ruc'] = {}
-    guiOverride['--disable-ruc']['bpa'] = False
 
     input_simulation_options.add_option('--run-deterministic-ruc',
```

Comment:
Why not just remove it instead of deprecating it?

Reply:
Only for current users of Prescient. Note that the test scripts currently utilize this option, so removing it would require updating any `*.txt` simulator script currently in-use.

I could probably be talked into removing it.